mlxtend/plotting/plot_confusion_matrix.py

```diff
@@ -26,29 +29,63 @@ def plot_confusion_matrix(conf_mat,
         Height and width of the figure
     cmap : matplotlib colormap (default: `None`)
         Uses matplotlib.pyplot.cm.Blues if `None`
-
+    colorbar : bool (default: False)
+        Shows a colorbar if True
+    show_absolute : bool (default: True)
+        Shows absolute confusion matrix coefficients if True.
+        At least one of  ``show_absolute`` or ``show_normed``
```

Comment:
Since the docstrings are parsed & rendered via markdown, could you use single back ticks? Otherwise the code formatting wouldn't work properly I think